src/services/updates/LibraryUpdateQueries.ts

```
@@ -189,14 +189,12 @@ const updateNovel = async (
   const oldTotalPages = await getStoredTotalPages(novelId);
 
   const novel = await fetchNovel(pluginId, novelPath);
-
   if (refreshNovelMetadata) {
     await updateNovelMetadata(pluginId, novelId, novel);
   } else if (novel.totalPages) {
     await updateNovelTotalPages(novelId, novel.totalPages);
-    await updateNovelTotalPages(novelId, novel.totalPages);
   }
-
+  console.log(novel.chapters);
```

ESLint

[error] Unexpected console statement.

(no-console)